nemo/collections/nlp/modules/common/megatron/megatron_init.py

```diff
@@ -27,6 +27,8 @@
     HAVE_APEX = True
 except (ImportError, ModuleNotFoundError):
 
+    from nemo.collections.nlp.parts.microbatch_calculator import ConstantNumMicroBatches, setup_microbatch_calculator
```

Comment:
What about set_logging_level symbol in except path?

Reply:
That will set the apex transformer log level, and we don't use apex in the except path